cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -94,6 +101,8 @@ class YamlConfig(
         }
         assertFileExists(xctestrunFile, "xctestrunFile")
 
+        for (device in devices) assertIosDeviceSupported(device)
```

Comment:
`devices.forEach`?

Reply:
I can change it if that's the preferred style. I'm still really new to Kotlin.

Reply:
```kotlin
devices.forEach { device -> assertIosDeviceSupported(device) }
```

i usually prefer named params, either way works